scripts/add-repo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Should we be running this with `set -e` since we aren't checking explicitly for failure cases?